ts/Extensions/DataGrouping.ts

```diff
@@ -923,6 +923,9 @@ const baseProcessData = seriesProto.processData,
         },
         ohlc: {
             groupPixelWidth: 5
+        },
+        heikinashi: {
```

Comment:
Please, add a comment to move this to `HeikinAshiSeries.ts` when dataGrouping will support any form of extending with new types